test-result-summary-client/src/TabularView/TabularView.jsx

```diff
@@ -612,7 +616,7 @@ export default class TabularView extends Component {
                 } );
         }
         const info = await tabularData.json();
-        const platformArray = [...new Set([...this.state.platforms,...(info.pop())])];
+        const platformArray = [...new Set([...this.state.platforms,...(info.pop().map(platform => getInfoFromBuildName(platform) ? getInfoFromBuildName(platform).platform : null))])];
```

Comment:
- Please use a parameter. Calling the function twice is not efficient.
- what is the `platform ` value that is passed into `getInfoFromBuildName ` ?

Reply:
- Created a function for it.
- Value of the `platform` is the whole name of the pipeline.